api-samples/idle/idle_simple/README.md

```diff
@@ -0,0 +1,19 @@
+# chrome.idle
```

Comment:
Since we only have one sample, can we move from `idle/idle_simple` to just `idle`?